src/app/routes/cpsAsset/getInitialData/convertToOptimoBlocks/blocks/include/index.js

```diff
@@ -92,7 +116,9 @@ const convertInclude = async includeBlock => {
     type,
     model: {
       href,
-      html: await fetchMarkup(buildIncludeUrl(href, includeType)),
+      html: isIncludeSupported(href, pathname)
+        ? await fetchMarkup(buildIncludeUrl(href, includeType), pathname)
+        : null,
```

Comment:
Reflecting the comment I've added to the issue, we should probably add an `ampUrl` param to model to allow us to construct the `amp-iframe` in the container: https://github.com/bbc/simorgh/issues/5989#issuecomment-641355816 

This would replace fetching the markup in the amp case.